docs/sphinx/developers/java-library.txt

```diff
@@ -37,55 +38,152 @@ repositories element can be defined at the top of your inheritance chain.
+      - `MIT-Style License`_
+    * - `Joda time v2.2 <http://github.com/JodaOrg/joda-time>`_
+      - joda-time:joda-time:2.2
+      - `Apache License v2.0`_
+    * - `JUnit v4.10 <http://www.junit.org>`_
+      - junit:junit:4.10
+      - `Common Public License v1.0`_
+    * - `Apache Log4j v1.2.17 <http://logging.apache.org/log4j/1.2>`_
+      - log4j:log4j:1.2.17
+      - `Apache License v2.0`_
+    * - `ImageJ v1.48s <http://imagej.net>`_
+      - net.imagej:ij:1.48s
+      - Public domain
+    * - `Assume NG v1.2.4 <http://github.com/hierynomus/assumeng>`_
+      - nl.javadude.assumeng:assumeng:1.2.4
+      - `Apache License v2.0`_
+    * - `Apache Velocity v1.6.4 <http://velocity.apache.org>`_
+      - org.apache.velocity:velocity:1.6.4
+      - `Apache License v2.0`_
+    * - `BeanShell v2.0b4 <http://www.beanshell.org>`_
+      - org.beanshell:bsh:2.0b4
+      - `Sun Public License / LGPL`_
+    * - `Hamcrest Core v1.1 <http://hamcrest.org/JavaHamcrest>`_
+      - org.hamcrest:hamcrest-core:1.1
+      - `BSD 3-Clause`_
+    * - `Objenesis v2.1 <http://objenesis.org>`_
+      - org.objenesis:objenesis:2.1
+      - `Apache License v2.0`_
+    * - `Perf4J v0.9.13 <http://www.perf4j.org>`_
+      - org.perf4j:perf4j:0.9.13
+      - `Apache License v2.0`_
+    * - `Native Library Loader v2.1.4 <http://github.com/scijava/native-lib-loader>`_
+      - org.scijava:native-lib-loader:2.1.4
+      - `Simplified BSD License`_
+    * - `SLF4J API v1.7.4 <http://www.slf4j.org>`_
+      - org.slf4j:slf4j-api:1.7.6
+      - `MIT License`_
+    * - `SLF4J LOG4J-12 Binding v1.7.6 <http://www.slf4j.org>`_
+      - org.slf4j:slf4j-log4j12:1.7.6
+      - `MIT License`_
+    * - `TestNG v6.8 <http://testng.org>`_
+      - org.testng:testng:6.8
+      - `Apache License v2.0`_
+    * - `SnakeYAML v1.6 <https://bitbucket.org/asomov/snakeyaml>`_
+      - org.yaml:snakeyaml:1.6
+      - `Apache License v2.0`_
+    * - `Jakarta ORO v2.0.8 <http://jakarta.apache.org/oro>`_
+      - oro:oro:2.0.8
+      - `Apache License v2.0`_
+    * - `Woolz v1.4.0 <http://www.emouseatlas.org/emap/analysis_tools_resources/software/woolz.html>`_
+      - woolz:JWlz:1.4.0
+      - `GPL v2`_
+    * - `Xalan Java Serializer v2.7.2 <http://xml.apache.org/xalan-j>`_
+      - xalan:serializer:2.7.2
+      - `Apache License v2.0`_
+    * - `Xalan Java v2.7.2 <http://xml.apache.org/xalan-j>`_
+      - xalan:xalan:2.7.2
+      - `Apache License v2.0`_
+    * - `Xerces2 Java Parser v2.8.1 <http://xerces.apache.org/xerces2-j>`_
+      - xerces:xercesImpl:2.8.1
+      - `Apache License v2.0`_
+    * - `XML Commons External Components XML APIs v1.3.04 <http://xml.apache.org/commons/components/external>`_
+      - xml-apis:xml-apis:1.3.04
+      - `Apache License v2.0`_
+
+
+.. _Apache License v2.0: http://www.apache.org/licenses/LICENSE-2.0.txt
+.. _MIT License: http://opensource.org/licenses/MIT
+.. _Simplified BSD License: https://github.com/scijava/native-lib-loader/blob/master/LICENSE.txt
```

Comment:
This is the equivalent of the 2-clause BSD license so both links could be unified